components/pages/Messages/Messages.module.css

```diff
@@ -253,3 +257,100 @@
   width: 100%;
   align-items: center;
 }
+.likeIconContainer {
+  width: max-content;
+  padding: 6px;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  background-color: #efefef;
+  margin-left: 52px;
```

Comment:
I don't think you need this.

Reply:
Yea i do, because the message's section doesn't have padding it has empty divs  the size of the users thumbnail before the message text. I can either put margin in or use the same empty div with display flex and a gap, or I can just use higher margin.